.editorconfig

```diff
@@ -0,0 +1,24 @@
+# WARNING - THIS FILE WAS GENERATED BY THE dps-gradle-spring-boot GRADLE PLUGIN
```

Comment:
may be better to add this to .gitignore and then will be created fresh each time?